Specs/Scene/PickSpec.js

```diff
@@ -58,6 +60,20 @@ defineSuite([
         primitives.removeAll();
     });
 
+     // reset the scene such that the canvas is 10 pixels and the camera is set
+    //  such that the primitive only takes up a small area at the canvas center
+    function setupPickSizeTestScene()  {
+        scene = createScene({canvas: createCanvas(10,10) });
+        primitives = scene.primitives;
+        camera = scene.camera;
+        scene.mode = SceneMode.SCENE3D;
+        scene.morphTime = SceneMode.getMorphTime(scene.mode);
+
+        camera.setView({
+            destination : Rectangle.fromDegrees(-10.0, -10.0, 10.0, 10.0)
+        });
+    }
```

Comment:
Maybe just always construct `scene` with a 10x10 canvas in `beforeAll` and add the camera adjustment to the individual tests. This will avoid the scene creation boilerplate.